africanmathsinitiative/R-Instat
# Fixing the issue of multiple output in the duplicates dialog

@africanmathsinitiative/developers this is ready for review

I noticed a very minor bug in the Duplicates dialog where if you clicked reset and clicked "OK", then the `summary` code was run multiple times. This PR fixes this so that this should not be an issue.

To test:
* Load data
* Open dialog (Prepare > Check Data > Duplicates)
* Click "Reset"
* Click "OK"
* The `summary` output should now only be printed once
